pkgs/applications/editors/vscode/generic.nix

```diff
@@ -225,6 +228,10 @@ in
       --replace "/bin/bash" "${bash}/bin/bash"
     rm -rf "$packed"
 
+    # this fixes "Node Debugger:Attach to process"
+    substituteInPlace resources/app/extensions/ms-vscode.js-debug/src/extension.js \
+      --replace-fail "/bin/ps" "${procps}/bin/ps"
```

Comment:
That no longer matches and I can't find it in https://github.com/search?q=repo%3Amicrosoft%2Fvscode-js-debug%20%2Fbin%2Fps&type=code